orderedmap_test.go

```diff
@@ -304,6 +304,16 @@ func TestOrderedMap_Back(t *testing.T) {
 	})
 }
 
+func TestOrderedMap_Copy(t *testing.T) {
+	t.Run("ReturnsEqualButNotSame", func(t *testing.T) {
+		m := orderedmap.NewOrderedMap()
+		m2 := m.Copy()
+
+		assert.Equal(t, m, m2)
+		assert.NotSame(t, m, m2)
```

Comment:
This doesn't really test anything. You should mutate an item in `m2` and assert that the same item has not changed in `m`.

Reply:
Ahh, makes sense. I didn't consider the pointers to the elements 😄